purchases/src/main/kotlin/com/revenuecat/purchases/galaxy/conversions/StoreProductConversions.kt

```diff
@@ -94,19 +94,23 @@ private fun ProductVo.createPricingPhases(
         eligibilityPricings.contains(ELIGIBILITY_PRICING_FREE_TRIAL) &&
         eligibilityPricings.contains(ELIGIBILITY_PRICING_TIERED_PRICE)
     ) {
-        // We are temporarily not handling Tiered Pricing price phases when there is both a trial and a tiered
-        // price available
-        log(LogIntent.GALAXY_WARNING) {
-            GalaxyStrings.PARSING_INTRO_PRICING_PHASES_FOR_SUBS_TIERED_PRICING_NOT_SUPPORTED
-        }
-        this.createFreeTrialPricingPhase()?.let { pricingPhases.addFirst(it) }
+        this.createFreeTrialPricingPhase()?.let { pricingPhases.add(it) }
```

Comment:
Replacing `addFirst` with `add` to avoid some Android OS level availability requirements